blueocean-dashboard/src/main/less/connection-lost.less

```diff
@@ -0,0 +1,30 @@
+.errorscreen {
+    &.connection-lost {
+        background-color: @errorscreen-bg-color;
+    }
+    &.connection-ok {
+        background-color: @blueocean-blue;
+    }
+
+    &.connection-lost, &.connection-ok {
+        /*
+         * Lets ensure that the lost and ok messages are the same
+         * width Vs jumpy message box resizes.
+         */
+        .message-box {
+            min-width: 30em;
+            max-width: 30em;
+        }
+
+        /*
+         * For some reason, we needed to set the font-weight on the
+         * h3 used in the errorscreen when disconnected. If not, it
+         * appears more bold than it does if the same screen is
+         * shown when connected, plus when compared to the 404 screen.
+         * Weird :(
+         */
+        h3 {
+            font-weight: 400;
+        }
```

Comment:
But, I mean, it should not be needed at all !! It's not needed if I display this component when the connection is ok ... why is it needed when the connection is not ok? There are no resource loading errors ... the computed CSS rules are exactly the same (in the browser dev tools). It's weird.